bip-0000.mediawiki

```diff
@@ -0,0 +1,472 @@
+<pre>
+  BIP: ????
+  Layer: Applications
+  Title: codex32
```

Comment:
Would prefer a title that means something

Reply:
Yes.

Maybe something like `Codex32 format for Shamir Secret Shares of [BIP-0032] master seeds`.

Reply:
Sure. How about "codex32: a bech32-based seed encoding supporting Shamir Secret Sharing and hand computation"?

Reply:
SGTM.

Reply:
Pushed.